### ISSUE TITLE
Running with my own data is not displayed with results

### ISSUE THREAD

**Issue by songhan0209** [AntSimi/py-eddy-tracker#153]
I want to ask you this question,When I run the modified pet_sla_and_adt.py  with the nc file I generated, no results are displayed, as shown in the following two pictures and pet_sla.txt,Please help me see what the problem is！thank you!
![1](https://user-images.githubusercontent.com/80887191/177730412-37ea017f-25fb-4666-b55d-524311c588f6.png)
![2](https://user-images.githubusercontent.com/80887191/177730445-ff3c9836-29f9-411e-a38c-f1060b745a85.png)
[pet_sla.txt](https://github.com/AntSimi/py-eddy-tracker/files/9061884/pet_sla.txt)

**Comment by songhan0209** [comment]
My nc file is as follows
[sla-2015-3.zip](https://github.com/AntSimi/py-eddy-tracker/files/9062104/sla-2015-3.zip)

**Comment by AntSimi** [comment]
Maybe try detection on whole file

**Comment by songhan0209** [comment]
When I use all the data, the result a_sla and the result of SLA picture are shown in the following figure, when I generate the SLA picture ,I modify diaplay parametersstill  to "m = g.display(ax, "sla_raw")" remove vmin and vmax .The remaining result part is the same as above, without a demonstration of the eddy curve.
![3](https://user-images.githubusercontent.com/80887191/177762529-1ce37953-76ba-4b45-b466-4ddf598a9513.png)
![4](https://user-images.githubusercontent.com/80887191/177762589-97f15b38-209f-4226-99d2-d8d533c9a314.png)
![5](https://user-images.githubusercontent.com/80887191/177763948-46566467-e0d0-4822-a764-2cdf55a848bf.png)

**Comment by songhan0209** [comment]
yes,I removed this options .I have another question,Whether this result error is related to the data order of magnitude, because my SLA is simulated.

**Comment by songhan0209** [comment]
when I use another  real download data from HYCOM,This error occurred like:  No extrema found in contour of 4pixels in level 0.634000,the result like this picture.I don't know why this.
![1](https://user-images.githubusercontent.com/80887191/177799974-5fe40390-401a-421e-bcd2-7d9d4d845438.png)

**Comment by songhan0209** [comment]
I never modify the previous files like grid.py,eddy_feature.py...

**Comment by AntSimi** [comment]
`No extrema found in contour of 4pixels` this warning is catch when all pixel in contour have same value

When i do a quick ncview on your sla file, i didn't see eddies
![image](https://user-images.githubusercontent.com/36040805/177804882-f534d4e8-6ca6-496a-86f6-cbcc694b2a87.png)

**Comment by songhan0209** [comment]
but why the top of picture show 4 Anticyclonic eddies ,9 cyclonic eddies?

**Comment by AntSimi** [comment]
> but why the top of picture show 4 Anticyclonic eddies ,9 cyclonic eddies?

Maybe you don't look at the right place, unzoom your plot or/and look at eddies coordinates
